app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    stored_results = {}
+
+    def inner(*args) -> Callable:
```

Comment:
The type annotation for the return type of `inner` should be `Any` instead of `Callable`